config/routes.rb

```diff
@@ -43,4 +40,20 @@
       resources :group_assignments, path: 'group-assignments'
     end
   end
+
+  namespace :stafftools do
+    mount Sidekiq::Web  => '/sidekiq'
```

Comment:
Hmmm. I'm really not a fan of specifying auth logic at the route level, but that seems to be the recommendation from Sidekiq: https://github.com/mperham/sidekiq/wiki/Monitoring#restful-authentication-or-sorcery